test/libsolidity/smtCheckerTests/operators/delete_function_type_3.sol

```diff
@@ -0,0 +1,19 @@
+contract C {
+    function()[] arr;
+    function() f;
+
+    constructor() {
+        arr.push();
+        arr.push();
+        arr[1] = f;
+        delete arr[1];
+        assert(arr[1] == f); // should fail for now because function pointer comparisons are not supported
+    }
+}
+// ----
+// Warning 8364: (82-90): Assertion checker does not yet implement type function (function ()[] storage pointer) returns (function ())
+// Warning 8364: (102-110): Assertion checker does not yet implement type function (function ()[] storage pointer) returns (function ())
+// Warning 7229: (172-183): Assertion checker does not yet implement the type function () for comparisons
+// Warning 6368: (149-155): CHC: Out of bounds access happens here.
+// Warning 6368: (172-178): CHC: Out of bounds access happens here.
```

Comment:
This happens because of the assignment `arr[1] = f`, considering `f` is of function type. It doesn't happen for value types.